services/callback/src/main/java/app/coronawarn/server/services/callback/controller/CallbackController.java

```diff
@@ -40,10 +43,17 @@ public CallbackController(FederationBatchInfoService federationBatchInfoService)
    */
   @GetMapping(value = CALLBACK_ROUTE, params = {"batchTag!="})
   @Timed(description = "Time spent handling callback.")
-  public ResponseEntity<Void> handleCallback(@RequestParam(required = true) String batchTag,
+  public ResponseEntity<Void> handleCallback(@RequestParam String batchTag,
```

Comment:
why is the batchTag not required anymore?

Reply:
@hilmarf required = true is the default case